### PR TITLE
Deploy release notes restructure

### PR DESCRIPTION
**Pull request by jonat123**

### CODE REVIEW

**Comment by sofietoft** [comment]
I like this way of structuring the release notes!
A few notes:

- Would be good with an introduction before all the expendables. Like, for how to read the article - first identify which major version you're using and then find the version in there.
- It says "Umbraco Deploy" a lot - as we're already on the Umbraco Deploy docs site, I don't think it's necessary to mention the product name in every expandable!
- A little note about the Umbraco Deploy Contrib product would be nice!
- The Deploy Contrib notes don't need to be in an expandable, as it kinda just duplicates the heading 🤔 or what do you think?

**Comment by jonat123** [comment]
@sofietoft Good points! 

I'll look into it :)

File: 10/umbraco-deploy/release-notes.md [hunk 1]
```diff
@@ -130,74 +173,163 @@ We've listed here all changes going back to March 2021 for Deploy 4 and above. F
...
+
+### [Deploy 4.6.2](https://github.com/umbraco/Umbraco.Deploy.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F4.6.2) (August 16th 2022)
+
+*   Aligned Git URL displayed in backoffice with that in Cloud Portal (V4 only) [#136](https://github.com/umbraco/Umbraco.Deploy.Issues/issues/136)
+*   Fixed issue with deployment of root node value for Umbraco Forms's "save as Umbraco node" workflow [#133](https://github.com/umbraco/Umbraco.Deploy.Issues/issues/133)
+*   Fixed incorrect availability of workspace restore in production environment (V9 and V10 only)
+*   Added close button to "Transfer now" dialog
+*   Resolved registration of deployable types to support configuration for "backoffice edition".
+
+### [Deploy 4.6.1,](https://github.com/umbraco/Umbraco.Deploy.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F4.6.1) (July 12th 2022)
```
Comment (eshanrnh) [inline 10/umbraco-deploy/release-notes.md:270]:
```suggestion
### [Deploy 4.6.1](https://github.com/umbraco/Umbraco.Deploy.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F4.6.1) (July 12th 2022)
```

File: 10/umbraco-deploy/release-notes.md [hunk 2]
```diff
@@ -130,74 +173,163 @@ We've listed here all changes going back to March 2021 for Deploy 4 and above. F
...
+*   Fixed typo in UI [#113](https://github.com/umbraco/Umbraco.Deploy.Issues/issues/113)
+*   Ensured signature refresh on Data Type move into or out of folder [#125](https://github.com/umbraco/Umbraco.Deploy.Issues/issues/125)
+*   Fixed selection of workspace for compare dialog
+*   Optimized existence checks in connectors
+*   Restore missing partial restore option in content and media tree roots (V9+)
+*   Fixed extract trigger URL in PowerShell script distributed with Deploy On-Premise (V9+)
+*   Improved deserialization of exceptions for clearer reporting (V9+)
+
+
+### [Deploy 4.6.0](https://github.com/umbraco/Umbraco.Deploy.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F4.6.0) [and](https://github.com/umbraco/Umbraco.Forms.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F8.12.0)  (April 26th 2022)
```
Comment (eshanrnh) [inline 10/umbraco-deploy/release-notes.md:284]:
```suggestion
### [Deploy 4.6.0](https://github.com/umbraco/Umbraco.Deploy.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F4.6.0) [and] [Deploy 8.12.0](https://github.com/umbraco/Umbraco.Forms.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F8.12.0)  (April 26th 2022)
```

**Review by eshanrnh — COMMENTED** [review]
Just noticed two glaring mistakes while viewing it on GitBook.. 🙈

File: 10/umbraco-deploy/release-notes.md (hunk 2 above)
Comment (jonat123) [reply 10/umbraco-deploy/release-notes.md:284]: Thanks for the suggestion :D It seems it is a link to a forms issue, so I'll just remove it :D

**Comment by jonat123** [comment]
@sofietoft I've updated the PR based on your feedback :) 

Let me know what you think! :D

File: 10/umbraco-deploy/release-notes.md [hunk 3]
```diff
@@ -8,15 +8,15 @@ We've listed here all changes going back to March 2021 for Deploy 4 and above. F
 
 ## Release History
 
-### [Deploy 4.7.3](https://github.com/umbraco/Umbraco.Deploy.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F4.7.3) (February 21st 2023)
+In the below section, you can find the release notes for each major version of Umbraco Deploy.
 
-*   All items listed under the 9.5.3, 10.1.3 and 11.0.1 releases (other than those indicated as only applying to higher versions).
+Under each dropdown for the major versions, you can find details about each new minor versions released for the specific major.
```
Comment (sofietoft) [inline 10/umbraco-deploy/release-notes.md:13]:
```suggestion
Under each major version, you can find details about minor and patch releases for the specific major.
```

File: 10/umbraco-deploy/release-notes.md [hunk 4]
```diff
@@ -8,15 +8,15 @@ We've listed here all changes going back to March 2021 for Deploy 4 and above. F
 
 ## Release History
 
-### [Deploy 4.7.3](https://github.com/umbraco/Umbraco.Deploy.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F4.7.3) (February 21st 2023)
+In the below section, you can find the release notes for each major version of Umbraco Deploy.
```
Comment (sofietoft) [inline 10/umbraco-deploy/release-notes.md:11]:
```suggestion
In this article, you can find release notes for each release of Umbraco Deploy.
```

File: 10/umbraco-deploy/release-notes.md [hunk 5]
```diff
@@ -32,33 +32,23 @@ We've listed here all changes going back to March 2021 for Deploy 4 and above. F
...
 *   Tidied up initialization markers [#102](https://github.com/umbraco/Umbraco.Deploy.Issues/issues/102)
+  
+</details>
 
-### Deploy Contrib 9.1.1, 10.1.1 and 11.0.1 (February 14th 2023)
+<details>
 
-*   Block grid editor support and fix for null reference exception (10+)
-*   Further caching for deploy operations.
+<summary>Umbraco Deploy Version 10</summary>
```
Comment (sofietoft) [inline 10/umbraco-deploy/release-notes.md:40]: It was actually these headings I was referring to 😅 

```suggestion
<summary>Version 10</summary>
```

**Review by sofietoft — COMMENTED** [review]
Just a couple more comments and suggestions.
The same can be applied to version 11.

File: 10/umbraco-deploy/release-notes.md (hunk 5 above)
Comment (jonat123) [reply 10/umbraco-deploy/release-notes.md:40]: Ahhh okay, I've removed it.

**Comment by jonat123** [comment]
@sofietoft all done :D 

Let me know if there is anything else :)

**Comment by sofietoft** [comment]
Looks good 💪 
Except I think you added version 11 to the Umbraco 10 version 😁

**Comment by jonat123** [comment]
ahhh I've removed v11 from the v10 version :D

File: 10/umbraco-deploy/release-notes.md [hunk 6]
```diff
@@ -8,15 +8,78 @@ We've listed here all changes going back to March 2021 for Deploy 4 and above. F
 
 ## Release History
 
-### [Deploy 4.7.3](https://github.com/umbraco/Umbraco.Deploy.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F4.7.3) (February 21st 2023)
+In this article, you can find release notes for each release of Umbraco Deploy.
```
Comment (sofietoft) [inline 10/umbraco-deploy/release-notes.md:11]: I thought that this was the first part of the article 🙈 
```suggestion
In this section, you can find release notes for each release of Umbraco Deploy.
```

File: 11/umbraco-deploy/release-notes.md [hunk 7]
```diff
@@ -32,93 +36,140 @@ We've listed here all changes going back to March 2021 for Deploy 4 and above. F
...
-### [Deploy 4.6.0](https://github.com/umbraco/Umbraco.Deploy.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F4.6.0) [and](https://github.com/umbraco/Umbraco.Forms.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F8.12.0) [9.4.0](https://github.com/umbraco/Umbraco.Deploy.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F9.4.0) (April 26th 2022)
+<details>
 
-*   All changes listed under 4.6.0-rc001 and 9.4.0-rc001
-*   Retained compact JSON formatting when transferring grid values
+<summary>Version 9</summary>
+
+### [9.5.3](https://github.com/umbraco/Umbraco.Deploy.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F9.5.3) (February 14th 2023)
+
+*   Applied various updates to improve performance and reduce likelihood of timeouts when transferring or restoring items in bulk [#128](https://github.com/umbraco/Umbraco.Deploy.Issues/issues/128) [#152](https://github.com/umbraco/Umbraco.Deploy.Issues/issues/152) [#148](https://github.com/umbraco/Umbraco.Deploy.Issues/issues/148) [#110](https://github.com/umbraco/Umbraco.Deploy.Issues/issues/110) [#106](https://github.com/umbraco/Umbraco.Deploy.Issues/issues/106)
```
Comment (jonat123) [inline 11/umbraco-deploy/release-notes.md:111]:
```suggestion
*   Applied updates to improve performance and reduce likelihood of timeouts when transferring or restoring items in bulk [#128](https://github.com/umbraco/Umbraco.Deploy.Issues/issues/128) [#152](https://github.com/umbraco/Umbraco.Deploy.Issues/issues/152) [#148](https://github.com/umbraco/Umbraco.Deploy.Issues/issues/148) [#110](https://github.com/umbraco/Umbraco.Deploy.Issues/issues/110) [#106](https://github.com/umbraco/Umbraco.Deploy.Issues/issues/106)
```

File: 11/umbraco-deploy/release-notes.md [hunk 8]
```diff
@@ -130,74 +181,158 @@ We've listed here all changes going back to March 2021 for Deploy 4 and above. F
...
-*   All changes from 4.5.0-rc001 and 9.3.0-rc001
-*   Fixed bug with deployments of templates involving alias renames
-
-### [Deploy 4.5.0-rc001](https://github.com/umbraco/Umbraco.Deploy.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F4.5.0) and [9.3.0-rc001](https://github.com/umbraco/Umbraco.Deploy.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F9.3.0) (February 15th 2022)
+### [9.3.0](https://github.com/umbraco/Umbraco.Deploy.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F9.3.0) (February 15th 2022)
 
 *   Content comparison dialog [#65](https://github.com/umbraco/Umbraco.Deploy.Issues/issues/65)
-*   Back-office deployment of members and member groups.
-*   Added support for deployment of history clean-up settings on document types (V4 only)
+*   backoffice deployment of members and member groups.
```
Comment (jonat123) [inline 11/umbraco-deploy/release-notes.md:194]:
```suggestion
*   Backoffice deployment of members and member groups.
```

File: 10/umbraco-deploy/release-notes.md [hunk 9]
```diff
@@ -130,74 +158,158 @@ We've listed here all changes going back to March 2021 for Deploy 4 and above. F
...
-*   All changes from 4.5.0-rc001 and 9.3.0-rc001
-*   Fixed bug with deployments of templates involving alias renames
-
-### [Deploy 4.5.0-rc001](https://github.com/umbraco/Umbraco.Deploy.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F4.5.0) and [9.3.0-rc001](https://github.com/umbraco/Umbraco.Deploy.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F9.3.0) (February 15th 2022)
+### [9.3.0](https://github.com/umbraco/Umbraco.Deploy.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F9.3.0) (February 15th 2022)
 
 *   Content comparison dialog [#65](https://github.com/umbraco/Umbraco.Deploy.Issues/issues/65)
-*   Back-office deployment of members and member groups.
-*   Added support for deployment of history clean-up settings on document types (V4 only)
+*   backoffice deployment of members and member groups.
```
Comment (jonat123) [inline 10/umbraco-deploy/release-notes.md:171]:
```suggestion
*   Backoffice deployment of members and member groups.
```

File: 10/umbraco-deploy/release-notes.md [hunk 10]
```diff
@@ -273,11 +369,76 @@ We've listed here all changes going back to March 2021 for Deploy 4 and above. F
...
+
+### Deploy Contrib 9.1.1 (February 14th 2023)
+
+*   Block grid editor support and fix for null reference exception (10+)
+*   Further caching for deploy operations.
+
+### Deploy Contrib 4.2.0, 9.1.0 and 10.1.0 (September 7th 2022)
+
+*   Used caching in value connectors to improve performance.
+*   Fixed issue with transfer of nested content string values using item templates (i.e. commencing with "{{") that were erroneously considered as JSON.
```
Comment (jonat123) [inline 10/umbraco-deploy/release-notes.md:431]:
```suggestion
*   Fixed issue with transfer of nested content string values using item templates (which is commencing with "{{") that were erroneously considered as JSON.
```

File: 11/umbraco-deploy/release-notes.md [hunk 11]
```diff
@@ -273,11 +392,76 @@ We've listed here all changes going back to March 2021 for Deploy 4 and above. F
...
+
+### Deploy Contrib 9.1.1 (February 14th 2023)
+
+*   Block grid editor support and fix for null reference exception (10+)
+*   Further caching for deploy operations.
+
+### Deploy Contrib 4.2.0, 9.1.0 and 10.1.0 (September 7th 2022)
+
+*   Used caching in value connectors to improve performance.
+*   Fixed issue with transfer of nested content string values using item templates (i.e. commencing with "{{") that were erroneously considered as JSON.
```
Comment (sofietoft) [inline 11/umbraco-deploy/release-notes.md:454]:
```suggestion
*   Fixed issue with transfer of nested content string values using item templates (for example commencing with "{{") that were erroneously considered as JSON.
```

File: 11/umbraco-deploy/release-notes.md [hunk 12]
```diff
@@ -273,11 +392,76 @@ We've listed here all changes going back to March 2021 for Deploy 4 and above. F
...
+
+### Deploy Contrib 4.1.2, 9.0.2, 10.0.1 (July 12th 2022)
+
+*   Optimized existence checks in connectors
+
+
+### Deploy Contrib 4.1.1 (April 26th 2022)
+
+*   Fixes issue with deployment of nested content properties within doc type grid editor values. [#82](https://github.com/umbraco/Umbraco.Deploy.Issues/issues/82)
```
Comment (sofietoft) [inline 11/umbraco-deploy/release-notes.md:463]:
```suggestion
*   Fixes issue with deployment of nested content properties within Document Type Grid Editor values. [#82](https://github.com/umbraco/Umbraco.Deploy.Issues/issues/82)
```

File: 10/umbraco-deploy/release-notes.md [hunk 13]
```diff
@@ -273,11 +369,76 @@ We've listed here all changes going back to March 2021 for Deploy 4 and above. F
...
+*   Fixed issue with transfer of nested content string values using item templates (which is commencing with "{{") that were erroneously considered as JSON.
+
+### Deploy Contrib 4.1.2, 9.0.2, 10.0.1 (July 12th 2022)
+
+*   Optimized existence checks in connectors
+
+
+### Deploy Contrib 4.1.1 (April 26th 2022)
+
+*   Fixes issue with deployment of nested content properties within doc type grid editor values. [#82](https://github.com/umbraco/Umbraco.Deploy.Issues/issues/82)
```
Comment (sofietoft) [inline 10/umbraco-deploy/release-notes.md:440]:
```suggestion
*   Fixes issue with deployment of nested content properties within Document Type Grid Editor values. [#82](https://github.com/umbraco/Umbraco.Deploy.Issues/issues/82)
```

**Review by sofietoft — CHANGES_REQUESTED** [review]
A few suggestions to resolve remaining vale warnings.

File: 11/umbraco-deploy/release-notes.md (hunk 12 above)
Comment (jonat123) [reply 11/umbraco-deploy/release-notes.md:463]: Doc type grid editor is the name of a package: https://our.umbraco.com/packages/backoffice-extensions/doc-type-grid-editor/ Which is the reason why I did not change the name 🤔 

Should it still be changed or?

Comment (sofietoft) [reply 11/umbraco-deploy/release-notes.md:463]: Ooh! Didn't realize 😄 Then ignore my suggestion!

**Comment by jonat123** [comment]
merging in!

**Pull request merged**